swo/mpt/cli/core/accounts/models.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseModel
+
+
+class Account(BaseModel):
+    id: str
+    name: str
+    type: str
+    secret: str
+    environment: str
+    is_active: bool = False
```

Comment:
I would suggest to just call it `active`. The `is_` prefix makes me think on a function.